frontend/src/services/userService.ts

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+import authHeader from "./authHeader";
+
+const API_URL = "http://localhost:8080/api/";
+
+const getAllTrips = () => {
+    return axios.get(API_URL + "trips", { headers: authHeader() });
```

Comment:
we should use an interceptor for axios to automatically attach the token with each request instead.